tempesta_fw/sock.c

```diff
@@ -350,7 +350,7 @@ ss_do_send(struct sock *sk, struct sk_buff **skb_head, int flags)
 	int size, mss = tcp_send_mss(sk, &size, MSG_DONTWAIT);
 	unsigned int mark = (*skb_head)->mark;
 
-	TFW_DBG3("[%d]: %s: sk=%p queue_empty=%d send_head=%p"
+	TFW_DBG3("[%d]: %s: sk=%pK queue_empty=%d send_head=%pK"
```

Comment:
The year in the copyright notice isn't updated in the file. Can we just bump the year in all the sources just to forget about the problem?

Reply:
Just didn't update copyright for the very small changes. I think at the end it's not a problem at all if you just forget to update a copyright - if you or a reviewer recall this, then just update the copyright, otherwise if you forget - not a big deal.